agent-checkins/financefee-255-immutable-export-operability-20260620.md

```
@@ -0,0 +1,110 @@
+baseline cost/value fidelity gap in the observation writer (`persistence/observation_store.py:1061-1066`,
+`net==gross`, null `total_value`); the data-defect/quarantine conditions
+(`project_m3_negative_signal_is_baseline_cost_gap`, `project_m3_exit_mark_base100_contamination`,
+`project_m3_positive_slices_quarantined`). **M3 remains HALT_DATA_DEFECT / no-promote; positive
+slices remain quarantined; no gate flipped.**
+
+## Commit note
+
+Per harness rule (commit only when asked), changes are left in the **working tree, uncommitted**.
+To fold into PR #323:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)